solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Features/Details/Scenes/LaunchDetails/Views/LaunchDetailsBodyView.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  LaunchDetailsBodyView.swift
+//  SpaceApp
+//
+//  Created by Razee Hussein-Jamal on 08/03/2022.
+//
+
+import SwiftUI
+
+struct LaunchDetailsBodyView: View {
+    struct Model {
+        let title: String
+        let description: String
+        let buttonTitle: String
+    }
+    
+    let model: Model
+    let buttonAction: () -> Void
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: .dsSpacing(.xSmall)) {
```

Comment:
[SUGGESTION]
The VStack indentation could be,
```
VStack(
     alignment: .leading,
    spacing: .dsSpacing(.xSmall)
)
```

Reply:
Thanks